docs/addons/proxy-router.config.js

```diff
@@ -27,12 +27,17 @@ module.exports = {
 module.exports.serve = {
   content: [__dirname],
   add: (app, middleware, options) => {
-    // since we're manipulating the order of middleware added, we need to handle
-    // adding these two internal middleware functions.
-    middleware.webpack();
-    middleware.content();
+    // this example shows how to properly add middleware _after_ the built-in
+    // webpack and content middleware. since middleware must be added at the
+    // end of the middleware stack, we have to call the default middlewares
+    // outselves
+    middleware.webpack().then(() =>
+      middleware.content({
+        index: 'index.htm',
+      })
+    );
 
-    // router *must* be the last middleware added
+    // this example assumes router must be added last
     app.use(router.routes());
```

Comment:
Looks like it must be added only after `webpack` and `content`...
maybe
```js
add: (app, middleware, options) => {
  middleware.webpack().then(() => {
    middleware.content({ index: 'index.htm' });
    app.use(router.routes());
  });
},
```
or
```js
async add(app, middleware, options) => {
  await middleware.webpack()
  await middleware.content()
  app.use(router.routes());
},
```

Reply:
you are the master of finding my typos! 🙇